pkgs/data/icons/hackneyed/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, fetchzip, stdenvNoCC, fetchFromGitLab, xcursorgen, imagemagick6, inkscape }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "hackneyed";
+  version = "0.8.2";
+
+  src = fetchFromGitLab {
+    owner = "Enthymeme";
+    repo = "hackneyed-x11-cursors";
+    rev = "${version}";
```

```suggestion
    rev = version;
```
pname is already a string. See the [nixpkgs-manual](https://nixos.org/manual/nixpkgs/stable/#sec-syntax)